### ISSUE TITLE
How to disable writing wrapper.log completely?

### ISSUE THREAD

**Issue by v20z** [winsw/winsw#882]
Hello Gentlemen.

Please advice is it possible and how  to completely disable creation and writing `service.wrapper.log` file?

Thank You.

**Comment by Anas-op** [comment]
yeah, i'm also requiring it, this file is creating issues with sizes

**Comment by emcodem** [comment]
+1 :)
